test/unit/storage/sfSessionStorageTest.php

```diff
@@ -26,6 +26,7 @@
 {
   $storage = new sfSessionStorage();
   $t->pass('->__construct() does not throw an exception when not provided with options');
+  $storage->shutdown();
```

Comment:
needed for PHP 8.0 & 8.1, or good to have?

Reply:
This is to avoid a warning generated since PHP 7.2 while running test : 
PHP Warning:  session_name(): Session name cannot be changed when a session is active
found details here : https://bugs.php.net/bug.php?id=76358

During this test, we create two new sfSessionStorage(); 
_which should not happen in a normal application_

Then the second instance of sfSessionStorage() , during the constructor, call again : 
`    session_name($sessionName);`
which trigger the Warning because we do not "shutdown()" the first one.

To fix it, I added a call to shutdown()

But shutdown was closing the current session by session_write_close(), but it do not allow another session to start later, because it do not change the static $sessionStarted boolean

So the next sfSessionStart() do not call session_start(), and all the remaining tests failed.

2 fixes: 
- call shutdown();
- allow to call another session_start()

I don't think it should break any application, as we should not start 2 sfSessionStorage() because it's handled by the framework only at factories bootstrap.